ips_client/ips_requests.py

```diff
@@ -37,10 +45,11 @@ def post_job(self, file: IO, service: ServiceType, out_type: OutputType,
 
         url = urllib.parse.urljoin(self.ips_url, f'/{service}/{self.API_VERSION}/{out_type}')
 
+        log.debug(f'Posting file to {url} with arguments {job_args} ...')
```

Comment:
I would prefer if the logs are somehow more consistent with the logs of e.g. the IPSOn-Controllers web server.

e.g.
```
"GET /health HTTP/1.1" 200 OK
```

So that we have the method name (GET, POST, ...) the path, and also the response.

Reply:
I've just realized that setting the debug level to `DEBUG` will already give us logs of the actual requests. Like this:
```
DEBUG:urllib3.connectionpool:Starting new HTTP connection (1): 192.168.77.10:21101
DEBUG:urllib3.connectionpool:http://192.168.77.10:21101 "POST /dnat/v3/images?region=european_union HTTP/1.1" 200 53
```